pandas/core/reshape/merge.py

```diff
@@ -488,7 +488,13 @@ def __init__(self, left, right, how='inner', on=None,
         self.right_on = com.maybe_make_list(right_on)
 
         self.copy = copy
+
```

Comment:
does this not break if ``suffixes=None`` is passed?

Reply:
it will indeed break, but do we expect to get `None` for suffixes? the default is `("_x", "_y")`. I could understand that people may want to use suffixes like: `(None, "_y")` if they want to keep the name of one of original columns... but use case for `suffixes=None` seems rare...